clients/javascript/lib/gen_types/EventGroup.ts

```diff
@@ -31,8 +31,4 @@ export type EventGroup = {
    * It allows to link a group of events to an outside entity
    */
   externalId: string | null
-  /**
-   * List of event IDs in the group
-   */
-  eventIds: Array<ID>
```

Comment:
No list of event IDs on the event group's side

Our flow will be
- Get event group via external ID
- Search calendar events by searching on the event group's id